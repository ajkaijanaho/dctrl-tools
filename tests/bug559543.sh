#!/bin/sh

# Test case due to Ryan Niebur <ryan@debian.org>.

set -e

$SORT_DCTRL -kpriority:nr,id:nr