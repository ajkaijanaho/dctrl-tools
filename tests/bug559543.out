id: 4
priority: 2

id: 2
priority: 2

id: 5
priority: 1

id: 3
priority: 1

id: 1
priority: 1

